src/routes/views.js

```diff
@@ -0,0 +1,12 @@
+const { Router } = require("express");
+
+const { renderHomepage, renderNotePage } = require("../controllers/views");
+
+const router = Router();
+
+// TODO check /vs*
+router.get("/", renderHomepage);
```

Comment:
Currently if we access any route other than `/` or `/notes` we get an error page with no matching route ( try going to `/note` for example). We can use the wildcard `/*` route to mean ANY path which would fix this, and mean any path which isn't specified will go to the index page. You will have to change the order of your routes if you do this, as the wildcard route will literally pick up anything, so it will need to be the last route listed (this also means that your `/api` routes would need to come first in your index routes file.